### PR TITLE
Fixed source muenchenstein_bl

### PR DESCRIPTION
**Pull request by r3turnNull**
Fixed the URL for the collection dates as it was changed by the municipality.

### CODE REVIEW

File: custom_components/waste_collection_schedule/waste_collection_schedule/source/muenchenstein_ch.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
+import json
+from datetime import datetime, timedelta
+
+import requests
+from bs4 import BeautifulSoup
+from waste_collection_schedule import Collection  # type: ignore[attr-defined]
+
+TITLE = "Abfallsammlung Münchenstein"
```
Comment (mampfes) [inline custom_components/waste_collection_schedule/waste_collection_schedule/source/muenchenstein_ch.py:8]: Remove Abfallsammlung - all swiss sources just use the name of the municipality.

File: custom_components/waste_collection_schedule/waste_collection_schedule/source/muenchenstein_ch.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+DESCRIPTION = "Source for Muenchenstein waste collection."
+URL = "https://www.muenchenstein.ch/abfuhrdaten"
+TEST_CASES = {
+    "Abfuhrkreis Ost": {"waste_district": "Abfuhrkreis Ost"},
+    "Abfuhrkreis West": {"waste_district": "492"},
+}
+
+
+IconMap = {
```
Comment (mampfes) [inline custom_components/waste_collection_schedule/waste_collection_schedule/source/muenchenstein_ch.py:17]: IconMap -> ICON_MAP

File: custom_components/waste_collection_schedule/waste_collection_schedule/source/muenchenstein_ch.py [hunk 3]
```diff
@@ -0,0 +1,77 @@
+import json
+from datetime import datetime, timedelta
+
+import requests
+from bs4 import BeautifulSoup
+from waste_collection_schedule import Collection  # type: ignore[attr-defined]
+
+TITLE = "Abfallsammlung Münchenstein"
+DESCRIPTION = "Source for Muenchenstein waste collection."
+URL = "https://www.muenchenstein.ch/abfuhrdaten"
```
Comment (mampfes) [inline custom_components/waste_collection_schedule/waste_collection_schedule/source/muenchenstein_ch.py:10]: Remove abfuhrdaten from URL.

**Review by mampfes — CHANGES_REQUESTED** [review]
This is a new source, right? Please add a md file in doc folder and run update_docu_links.py to update README.md and info.md

**Comment by dt215git** [comment]
muenchenstein was removed in 1.32 due to a broken source. I guess that means the md, readme, and info files also changed.

**Comment by mampfes** [comment]
> muenchenstein was removed in 1.32 due to a broken source. I guess that means the md, readme, and info files also changed.

Ah, thanks! That's why this sounded so familiar to me :-) 

In the meantime some refactorings happend, therefore they have to be retightened in the source code. I already added comments to the PR. And the docu has to be recovered and potentially adapted.

**Comment by mampfes** [comment]
I did the requested changes by myself.

**Comment by mampfes** [comment]
Thanks for your contribution.

**Pull request merged**